tools/osbuilder/node-builder/azure-linux/README.md

```diff
@@ -66,7 +66,12 @@ For Kata:
 ```
 sudo dnf -y install kata-packages-uvm-build
 pushd /opt/kata-containers/uvm/tools/osbuilder/node-builder/azure-linux
+
+# You can make both debug/release UVMs with `make uvm` or 
+# do a granular build by specifying a release/debug target
 sudo make uvm
+sudo make uvm-release
+sudo make uvm-debug
```

Comment:
I wouldn't add these 2 lines to the same script, that's confusing with the existing `make uvm`. Should be 2 new paragraphs IMO.

Reply:
Addressed, thanks!